internal/service/clouddiscovery/model_destination.go

```diff
@@ -34,14 +45,47 @@ var DestinationResourceSchemaAttributes = map[string]schema.Attribute{
 		Computed:            true,
 		MarkdownDescription: "Destination configuration.",
 	},
+	"created_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Timestamp when the object has been created.",
+	},
+	"deleted_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Timestamp when the object has been deleted.",
+	},
 	"destination_type": schema.StringAttribute{
 		Optional: true,
 		Computed: true,
 		Validators: []validator.String{
 			internalvalidator.StringNotNull(),
 			stringvalidator.OneOf("DNS", "IPAM/DHCP", "ACCOUNTS"),
 		},
-		MarkdownDescription: "Destination type: DNS or IPAM/DHCP or ACCOUNTS.",
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIf(
```

Comment:
Thie Plan modifier needs to work only when the destination type changes . In case , a new destination is added ,such as , IPAM (in POSt call ) -> IPAM +DNS in (PUT call ) , the plan modifier should not be triggered and update should take place instead

Reply:
As discussed let's not make it complicated. RequiresReplaceIfConfigured should be sufficient for now.